app/assets/src/components/visualizations/table/BaseTable.jsx

```diff
@@ -65,9 +65,10 @@ class BaseTable extends React.Component {
     });
   }
 
-  basicHeaderRenderer({ columnData, label }) {
+  basicHeaderRenderer({ dataKey, columnData, label }) {
```

Comment:
This actually isn't necessary to ensure the Project label is left-aligned, see the below comment